src/Foundatio.TestHarness/Caching/CacheClientTestsBase.cs

```diff
@@ -880,9 +874,8 @@ public virtual async Task MeasureSerializerComplexThroughputAsync()
         {
             await cache.RemoveAllAsync();
 
-            var start = SystemClock.UtcNow;
+            var sw = Stopwatch.StartNew();
```

Comment:
do we even need these?

Reply:
No, but it was doing the same thing before. We probably should log out the timing since the test is apparently testing throughput.